src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java

```diff
@@ -17,492 +17,202 @@
 
 package org.killbill.billing.plugin.adyen.api;
 
+import com.google.common.base.Strings;
```

Comment:
I almost forgot last one: We have `org.killbill.commons.utils.Strings` in `killbill-commons:killbill-utils`

Reply:
Is this a `0.24.x` change or would it work on `0.22.x` as well? This PR is for the plugin version that is `0.22.x` compatible.

Reply:
Hm.. That import only used for `isEmptyOrNull()`, yes? If so, then I propose to create private method with `// FIXME: ` comment that once we not support 0.22.x anymore, we should use killbill's `Strings` instead.